scripts/check_release_notes.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
```

Comment:
I recommend against `set -e` since bash does silent exits on errors. This makes it hard to debug.

```suggestion
set -u -o pipefail
```

Then make sure we have good error checking within the script.

Reply:
I usually go for "fail fast" in case the script doesn't  `have good error checking`, it's safer.
But I can change here as I assume the needed checks are there.
(also, nothing enforces this apparently https://github.com/search?q=repo%3Aprometheus%2Fprometheus+%22set+-e%22&type=code)